package.json

```diff
@@ -30,7 +30,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "2.7.1",
     "ts-node": "10.8.1",
-    "typescript": "5.0.2",
+    "typescript": "5.0.4",
```

Comment:
can you run `yarn` and include the automatic changes to `yarn.lock`? that needs to be updated to make sure CI and other machines run the same version

Reply:
actually I don't think this will break CI so I can do it after merging